docs/book/installation.rst

```diff
@@ -74,8 +95,41 @@ browser.
 
 .. tip::
 
-    You can also use the Symfony built-in webserver by running the ``php app/console server:run --env dev``
-    command and then accessing ``http://localhost:8000``.
+    We strongly recommend using the Symfony abuilt-in webserver by running the
+    ``php app/console server:run --env dev``
+    command and then accessing ``http://localhost:8000``
+
+.. note::
+
+    The localhost's 8000 port may be already occupied by some other process.
+    If so you should try other ports, like for instance:
+    ``php app/console server:run localhost:8081 --env dev``
```

Comment:
I would remove this sentence and write something like "Want to know more about using build in server, go to http://symfony.com/doc/current/cookbook/web_server/built_in.html"